src/App.js

```diff
@@ -0,0 +1,100 @@
+import React, {Component} from 'react';
+import _ from 'lodash';
+import * as apiClient from './http/apiClient';
+import {newList} from './factories';
+
+import ListsView from './components/ListsView';
+import ListView from './components/ListView';
+
+export default class App extends Component {
```

Comment:
The burning question in my mind is whether to use ES6 classes or React.createClass throughout this project...

Personally I use and prefer createClass throughout all of my other projects. I decided to go with the class syntax for this project mostly just to give it a shot...

I'm leaning towards gutting out the ES6 class usage here in favor of createClass. I feel like it might be more of a distraction to new comers than anything... Any compelling reason to stick with `class`?